actions/validate-payment/index.js

```diff
@@ -29,11 +29,16 @@ async function main(params) {
       return webhookErrorResponse(`Failed to verify the webhook signature: ${error}`);
     }
 
-    const { payment_additional_information: paymentInfo, payment_method: paymentMethod } = params;
+    let payload = params;
+    if (params.__ow_body) {
+      payload = JSON.parse(atob(params.__ow_body));
```

Comment:
It'd be good to add a comment like the other actions:
```suggestion
      // in the case when "raw-http: true" the body needs to be decoded and converted to JSON
      payload = JSON.parse(atob(params.__ow_body));
```
https://github.com/adobe/commerce-checkout-starter-kit/blob/main/actions/tax-calculation/index.js#L42C1-L42C94